lua/modules/uikit.lua

```diff
@@ -1788,6 +1788,16 @@ function createUI(system)
 		_textInputRefreshColor(node) -- apply initial colors
 
 		node.contentDidResizeSystem = function(self)
+			print(
```

Comment:
Logs form a world page (`Another test world`):
```
[INFO]Another test world 535 self._refresh: [Function 0000023AF8D7A4D0] Screen.Width: 1706.0 Screen.Height: 960.0
[INFO]Another test world 948 self._refresh: [Function 0000023AF8D7A4D0] Screen.Width: 1706.0 Screen.Height: 960.0
[INFO]Another test world 172 self._refresh: nil Screen.Width: 1706.0 Screen.Height: 960.0
[INFO]Another test world 703 self._refresh: [Function 0000023AF8D7A4D0] Screen.Width: 1706.0 Screen.Height: 960.0
[INFO]Another test world 227 self._refresh: [Function 0000023AF8D7A4D0] Screen.Width: 1706.0 Screen.Height: 960.0
[INFO]Another test world 495 self._refresh: nil Screen.Width: 1706.0 Screen.Height: 960.0
[INFO]Another test world 125 self._refresh: [Function 0000023AF8D7A4D0] Screen.Width: 1706.0 Screen.Height: 960.0
[TRACE]willResignActive
[INFO]Another test world 84 self._refresh: [Function 0000023AF8D7A4D0] Screen.Width: 0.0 Screen.Height: 0.0
[INFO]Another test world 390 self._refresh: nil Screen.Width: 0.0 Screen.Height: 0.0
[INFO]Another test world 278 self._refresh: nil Screen.Width: 0.0 Screen.Height: 0.0
[INFO]Another test world 369 self._refresh: nil Screen.Width: 0.0 Screen.Height: 0.0
[INFO]Another test world 984 self._refresh: nil Screen.Width: 0.0 Screen.Height: 0.0
[INFO]Another test world 536 self._refresh: [Function 0000023AF8D7A4D0] Screen.Width: 0.0 Screen.Height: 0.0
[TRACE]didBecomeActive
[TRACE]ðŸ–Œï¸ game renderer views changed (2560, 1440, 1.500000) using (3412, 1920) at render scale 2.000000
[INFO]Another test world 766 self._refresh: [Function 0000023AF8D7A4D0] Screen.Width: 1706.0 Screen.Height: 960.0
[INFO]Another test world 647 self._refresh: nil Screen.Width: 1706.0 Screen.Height: 960.0
[INFO]Another test world 768 self._refresh: nil Screen.Width: 1706.0 Screen.Height: 960.0
[INFO]Another test world 781 self._refresh: nil Screen.Width: 1706.0 Screen.Height: 960.0
[INFO]Another test world 823 self._refresh: nil Screen.Width: 1706.0 Screen.Height: 960.0
[INFO]Another test world 152 self._refresh: [Function 0000023AF8D7A4D0] Screen.Width: 1706.0 Screen.Height: 960.0
```
It is strange that the `_refresh` function is sometimes nil